src/goto-instrument/contracts/contracts.cpp

```diff
@@ -1260,3 +1259,34 @@ bool code_contractst::enforce_contracts(const std::set<std::string> &to_enforce)
   }
   return fail;
 }
+
+void code_contractst::find_static_locals(
+  const goto_functionst &functions,
+  const irep_idt &root_function,
+  assigns_clauset &assigns)
+{
+  auto cg =
+    call_grapht::create_from_root_function(functions, root_function, true);
+  auto g = cg.get_directed_graph();
```

Comment:
[nit] do we need to keep `cg` around? also, may be use a slightly more descriptive name for `g` and make it `const`?

```suggestion
  const auto call_graph = call_grapht::create_from_root_function(functions, root_function, true).get_directed_graph();
```
(would need to reformat ^)

Reply:
renamed as required